modules/ROOT/pages/external-configuration.adoc

```diff
@@ -40,15 +40,15 @@ Thus, it is important to change the configuration of microservices without updat
 
 == MicroProfile Config
 
-xref:reference:feature/mpConfig-1.4.adoc[MicroProfile Config] separates the configuration from code.
+xref:mpConfig-1.4.adoc[MicroProfile Config] separates the configuration from code.
 With MicroProfile Config, you can inject the external configuration into services in the containers without repackaging them.
 
-Applications can use MicroProfile Config as a single API to retrieve configuration information from different sources such as link:reference:config/server-configuration-overview.adoc[system properties, system environment variables], properties files, XML files, or data sources.
+Applications can use MicroProfile Config as a single API to retrieve configuration information from different sources such as link:/docs/21.0.0.2/reference/config/server-configuration-overview.html[system properties, system environment variables], properties files, XML files, or data sources.
 In MicroProfile Config, these data sources are called config sources.
-The same configuration property can be defined in link:https://openliberty.io/guides/microprofile-config-intro.html#configuring-with-the-properties-file[multiple config sources].
+The same configuration property can be defined in link:/guides/microprofile-config-intro.html#configuring-with-the-properties-file[multiple config sources].
 
-With MicroProfile Config, you can https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_simple_dependency_injection_example[inject an individual property into a class], or https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_aggregate_related_properties_into_a_cdi_bean[inject multiple properties in one CDI bean].
-You can also retrieve values from configuration https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_simple_programmatic_example[without using a CDI bean].
+With MicroProfile Config, you can link:/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_simple_dependency_injection_example[inject an individual property into a class], or link:/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#_aggregate_related_properties_into_a_cdi_bean[inject multiple properties in one CDI bean].
```

Comment:
to link to an external site, you don't need `link:` but you do need `https://download.eclipse.org/` or whatever the URL for the external site is. I think the way you had these should be correct if those are working URLs